docs/guides/getting-started-orchestrate-microservices.md

```diff
@@ -45,6 +39,12 @@ Start by designing your automated process using BPMN. This guide introduces you
 8. Start a new process instance by clicking on the blue **Start instance** button.
 9. To the right of the two blue buttons, click the Application icon (honeycomb icon) button next to the **Start instance** button. Navigate to Operate to see your process instance with a token waiting at the service task by clicking **View process instances**.
 
+### Create a cluster
```

Comment:
To confirm, is it correct that here we say you can essentially start an instance without having a cluster, but you cannot deploy a diagram without having a cluster?

Reply:
@christinaausley If you try to start an instance **or** deploy a diagram without a cluster you'll see a modal with a link to create a new cluster. Can you confirm this in your organization? It looks like this is in production, but I'm having trouble confirming (which prompted my message on this PR).